.github/workflows/_ios-build-test.yml

```diff
@@ -121,15 +114,13 @@ jobs:
         if: inputs.ios-platform == 'SIMULATOR'
         run: |
           # shellcheck disable=SC1091
-          source "${RUNNER_TEMP}/anaconda/bin/activate"
+          source "${MINICONDA_INSTALL_PATH}/bin/activate" "${RUNNER_TEMP}/${CONDA_ENV}"
           # use the pytorch nightly build to generate models
           pip3 install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
           # generate models for differnet backends
           cd "${GITHUB_WORKSPACE}/ios/TestApp/benchmark"
           mkdir -p ../models
           if [ "${USE_COREML_DELEGATE}" == 1 ]; then
-            pip install coremltools==5.0b5 protobuf==3.20.1
-            pip install six==1.16.0
             python coreml_backend.py
```

Comment:
Also add `${CONDA_RUN}` here

Reply:
`${CONDA_RUN}` prefix was added for the corresponding lines. No sign of Ruby issue for now, all CI tests are green, except for `libtorch-cpu-failed` and `linux-bionic-cuda11.6` which should not be related.

Reply:
libtorch builds are unrelated